<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindForm</class>
 <widget class="QDialog" name="FindForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_Find">
      <attribute name="title">
       <string>Find</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="find_lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="find_Button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="parameter_groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="direction_label">
              <property name="text">
               <string>Direction:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="direction_comboBox">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="register_checkBox">
              <property name="text">
               <string>Сase sensitive</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="word_checkBox">
              <property name="text">
               <string>Whole words</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="more_Button">
           <property name="text">
            <string>&lt;&lt; More options</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Replace">
      <attribute name="title">
       <string>Replace</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="find_lineEdit_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="find_Button_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="parameter_groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="direction_label_3">
              <property name="text">
               <string>Direction:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="direction_comboBox_3">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QCheckBox" name="register_checkBox_3">
              <property name="text">
               <string>Сase sensitive</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="word_checkBox_3">
              <property name="text">
               <string>Whole words</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="more_Button_3">
           <property name="text">
            <string>&lt;&lt; More options</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLineEdit" name="replace_lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="replace_Button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>more_Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>parameter_groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>more_Button_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>parameter_groupBox_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
